server/migrations/v2.17.3-share-add-isdownloadable.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { DataTypes } = require('sequelize')
+
+/**
+ * @typedef MigrationContext
+ * @property {import('sequelize').QueryInterface} queryInterface - a Sequelize QueryInterface object.
+ * @property {import('../Logger')} logger - a Logger object.
+ *
+ * @typedef MigrationOptions
+ * @property {MigrationContext} context - an object containing the migration context.
+ */
+
+/**
+ * This migration script adds the isDownloadable column to the mediaItemShares table.
+ *
+ * @param {MigrationOptions} options - an object containing the migration context.
+ * @returns {Promise<void>} - A promise that resolves when the migration is complete.
+ */
+module.exports = {
```

Comment:
Migrations should keep the same format of using individual functions `up` and `down` for consistency. The JSDoc types also aren't going to work when they are above the export like that.
If you're not using an editor that can use JSDocs I highly recommend it.

The other migrations have a log for BEGIN/END on up and down which we should keep consistent